client/lib/checkout/tax-placeholders.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get, has, isString, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:tax-placeholders' );
+
+export function maybeInjectTaxPlaceholdersIntoPurchase( purchase ) {
+	const purchasePriceText = get( purchase, 'price_text' );
+	if (
+		config.isEnabled( 'tax-placeholders' ) &&
+		! has( purchase, 'tax_text' ) &&
+		isString( purchasePriceText ) &&
+		! purchasePriceText.match( /include/i ) // "Included with plan"
+	) {
+		debug( 'injecting taxAmount into purchase', purchase );
+		return Object.assign( purchase, {
+			tax_amount: purchase.amount / 10,
+			tax_text: purchasePriceText.replace( /(?<!\d|\.)(\d+)(\d)(?:\.(\d*)\d)?/, '$1.$2$3' ),
```

Comment:
<img width="292" alt="screen shot 2018-12-18 at 6 35 24 pm" src="https://user-images.githubusercontent.com/6458278/50138956-5b62fb00-02f4-11e9-826a-0f1ad5f4590d.png">

Should we add the zero if necessary?

It seems quite clever, but is there a reason why we can't somehow use `tax_amount` (purchase.amount / 10) and `refund_currency_symbol` if needed? What if `purchase.amount / 10` varies?

I guess this is a first iteration, but might be worth unit testing this module in the future.

Reply:
When it's ready the backend should give us all the numbers as well as a pre-formatted string to display, so these are just placeholders to give us something to render until then, and it's fine if they're "wrong".

This whole file will go before launch.

I'll add a comment to the file to that effect.

Reply:
Oh, to address the "clever", I just want a string, and this regex has gotten a little overdone incrementally. If the regex "misses" you end up with "$18(+$18 tax)", which is fine for our purposes, and I could just pull it out.

The convention is to receive the preformatted number from the backend so I'm going to see how it looks first, but there's a good chance I'll end up doing the formatting in the front end like you suggest.